From fe52cfb512a2eacd7f7ed76e1c6687217eb064f1 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 14 Dec 2017 14:53:54 +0800
Subject: [PATCH 3106/5242] MLK-17215-2 ARM: dts: imx7ulp: add
 'phy-ref-clkfreq' property for mipi dsi

commit  7a473e51c83dfe2ccd65dacc27610a01da9ec763 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'phy-ref-clkfreq' property for 'mipi_dsi' node on
imx7ulp board.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index c8ca3e9..a581a307 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -493,6 +493,7 @@
 			clocks = <&clks IMX7ULP_CLK_DSI>;
 			clock-names = "mipi_dsi_clk";
 			data-lanes-num = <2>;
+			phy-ref-clkfreq = <24000000>;
 			max-data-rate  = <800000000>;
 			sim = <&sim>;
 			status = "disabled";
-- 
1.7.9.5

